aptos-move/aptos-release-builder/src/components/mod.rs

```diff
@@ -282,7 +284,9 @@ impl Default for ReleaseConfig {
     fn default() -> Self {
         ReleaseConfig {
             testnet: true,
-            framework_release: true,
+            framework_release: Some(FrameworkReleaseConfig {
+                bytecode_version: 5,
```

Comment:
does this need to be updated whenever we do framework release?

Reply:
This is the oldest file format version so technically it should always be supported. It would work until aptos framework start to use new file format features. Practically tho the default shouldn't be used for production and we should always provide the bytecode version that we would support.